index.bs

```diff
@@ -4814,22 +4814,22 @@ Updating Registry Tables</h4>
 	(i.e. [[#correction-classes|Class 5 changes]])
 	can be made by re-publishing the [=technical report=] that contains the affected table,
 	without needing to satisfy any other requirements for the publication
-	(not even Working Group consensus, unless this is required by the [=registry definition=]).
+	(not even consensus, unless this is required by the [=registry definition=]).
 	Such [=registry changes=] do not trigger new [=Advisory Committee Reviews=],
 	nor Exclusion Opportunities,
 	and do not require verification via an [=update request=],
 	even for [=technical reports=] at maturities where this would normally be expected.
 	Such publications can be made
-	even in the absence of a [=Working Group=] chartered to maintain the registry
+	even in the absence of a group chartered to maintain the registry
```

Comment:
```suggestion
	even in the absence of a [=group=] chartered to maintain the registry
```

Reply:
`[=groups=]` also refers to the AB, TAG, and AC, which I don't think is appropriate in the context of this sentense

Reply:
@fantasai , are you OK with closing this for the reason given above?

Reply:
Can a Community Group charter not include maintaining a registry?

Reply:
Community groups are not covered by the process at all, consensus doesn't apply (at least not necessarily, though they can opt into it), AC reviews don't apply, charters can be re-written on the fly without external validation… In general, CGs don't do any publication of any kind of technical report.

Now, if some other group has established a registry, and got the registry definitions properly approved, those definitions can designate anyone to be a custodian, and following the rules laid out in the definitions, make updates to the registry data. "Anyone" can be a CG. But a CG itself cannot establish the registry.

Reply:
Does it matter if a custodian is a group with a Charter but the Charter doesn't mention the custodianship role? I would say it's a "nice to have" that the Charter mentions it, but not essential.

Reply:
I would agree it doesn't matter, but I'm not sure why that's relevant.

Reply:
The only relevance is: if we thought it _is_ important that custodians with charters include their custodianship in their charter, then I would propose opening an additional issue to address that; since we seem to be in agreement that it is not important, then I would suggest that we don't need to do anything in response to @TallTed 's [question](https://github.com/w3c/process/pull/972#discussion_r1924307185). But @TallTed may have been going somewhere else with that, I don't know.